File: Server/src/commonStructures/serverEventType.ts
/**
 * All the possible events that can be sent from the server to the client
 */
enum ServerEventType{
    JOINED,
    GAME_OVER,
    LEFT,
    SPAWN,
    SPAWN_UNMANAGED,
    DESTROY,
    PICKUP,
    DROP,
    HEXAGON_PROPERTY_CHANGED,
    INITIAL_GAME_STATE,
    GAME_STATE_STREAM,
}

export default ServerEventType;

File: Assets/Scripts/Network/CommonStructures/ServerEventType.cs
namespace Network
{
    /// <summary>
    /// All the possible events that can be sent from the server to the client
    /// </summary>
    public enum ServerEventType
    {
        JOINED,
        GAME_OVER,
        LEFT,
        SPAWN,
        SPAWN_UNMANAGER,
        DESTROY,
        PICKUP,
        DROP,
        HEXAGON_PROPERTY_CHANGED,
        INITIAL_GAME_STATE,
        GAME_STATE_STREAM
    }
}